src/test/java/hudson/plugins/throttleconcurrents/ThrottleConcurrentTest.java

```diff
@@ -25,6 +26,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Ignore("Depends on a newer version of Guava than can be used with Pipeline")
```

Comment:
Actually, on second thought, this is basically covered by `ThrottleStepTest`, isn't it?